lambda_functions/app.py

```diff
@@ -237,31 +244,34 @@ def get_all_users(event, context):
         }
 
 
-def get_user_by_id(event, context):
```

Comment:
I think other methods rely on get_user_by_id. Wouldn't it make sense to leave get_user_by_id and create a separate get_active_user lambda?

Reply:
Nevermind, I confused the methods in app.py and operations.py